### PR TITLE
Fix job_task_id not found if db doesnt have the field    🐛  🚑️

### PR DESCRIPTION
**Pull request by wvangeit**
## What do these changes do?

A new field was added to the function jobs to store the celery task id. However, this is not present in existing entities, which generates 500 errors in production atm.
I'm creating this small PR to be applied as a hotfix to handle that case. 
By using get() in the dictionary the value will default to None.

## How to test

Test code on deployment that has old function job entities

## Dev-ops

No changes
To be hotfixed in production.

### CODE REVIEW

File: services/web/server/src/simcore_service_webserver/functions/_functions_service.py [hunk 1]
```diff
@@ -859,9 +859,9 @@ def _decode_functionjob(
             inputs=functionjob_db.inputs,
             outputs=functionjob_db.outputs,
             project_job_id=functionjob_db.class_specific_data["project_job_id"],
-            job_creation_task_id=functionjob_db.class_specific_data[
+            job_creation_task_id=functionjob_db.class_specific_data.get(
```
Comment (pcrespov) [inline services/web/server/src/simcore_service_webserver/functions/_functions_service.py:862]: I would add tests to cover none here

**Review by sanderegg — APPROVED** [review]
